sktime/forecasting/base/_base.py

```diff
@@ -2455,13 +2455,14 @@ def _predict_moving_cutoff(
         # iterate over data
         for new_window, _ in cv.split(y):
             y_new = y.iloc[new_window]
+            X_new = X.iloc[new_window]
```

Comment:
X may not always be a pandas DataFrame, it can be None as well. You should check first if it is None or not, and try to access specific window only if it's not None. Otherwise, keep X_new as None.

This is just the explanation of why the test is failing, not a review.

Reply:
@yarnabrina, Thank-you, I will make the necessary changes.